devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,32 @@
+from devolving_music.lib.elo_scoring import *
+class Song_Scores():
+    
+    def __init__(self,Song):
+        self._song_submision=Song
+        self._energy_score=0
+        self._quality_score=0
+        self._post_peak_score=0
+        self._info_score=0
+
+    @property
+    def energy_score(self):
+        return self._energy_score
+    @property
+    def quality_score(self):
+        return self._energy_score
+    @property
+    def post_peak_score(self):
+        return self._post_peak_score
+    @property
+    def info_score(self):
+        return self._info_score
```

Comment:
I appreciate you using the `@property` decorator. I was really only insisting on it for methods that contain actual computation as a way to cache their values, but this is fine.